process/peer/process_test.go

```diff
@@ -1246,9 +1237,9 @@ func TestValidatorStatisticsProcessor_CheckForMissedBlocksWithRoundDifferences(t
 				want: result{
 					decreaseValidatorValue: intValidatorProbability,
 					decreaseLeaderValue:    intLeaderProbability,
-					tempRating: rater.StartRating -
-						intLeaderProbability*rater.DecreaseProposer -
-						intValidatorProbability*rater.DecreaseValidator,
+					tempRating: uint32(int32(rater.StartRating) +
```

Comment:
is it possible to overflow? 
e.g uint32(negative value)

Reply:
it is just for testing purposes and with the current configuration it shouldn't happen